crates/std/src/uncategorized/asset_url.rs

```diff
@@ -236,7 +236,10 @@ impl AbsAssetUrl {
         Ok(Self(self.to_download_raw_url(assets)?))
     }
     fn to_download_raw_url(&self, assets: &AssetCache) -> Result<Url, url::ParseError> {
-        self.to_download_url_with_base(&ContentBaseUrlKey.get(assets))
+        let content_url = ContentBaseUrlKey.get(assets);
+        log::info!("Converting {self} to a download url. Found {content_url:?}");
```

Comment:
Log necessary?

Reply:
Nope, just missed it trying to figure out how it resolved an asset url to the server address